ioxd_router2/src/lib.rs

```diff
@@ -151,6 +151,7 @@ pub async fn create_router2_server_type(
     object_store: Arc<DynObjectStore>,
     write_buffer_config: &WriteBufferConfig,
     query_pool_name: &str,
+    request_limit: usize,
```

Comment:
```suggestion
    concurrent_request_limit: usize,
```

Maybe would make it more self-describing? Not required